cmd/mimir/Dockerfile.distroless

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: AGPL-3.0-only
+
+ARG        BASEIMG=gcr.io/distroless/static-debian12
```

Comment:
Perhaps we should include link to https://github.com/GoogleContainerTools/distroless/blob/main/base/README.md to explain what `static` and `base-nossl` (used by Makefile) mean.